src/App.php

```diff
@@ -130,7 +132,7 @@ public function run($request = null, $response = null)
         $httpFlow = $this->getEventManager();
 
         try {
-            $routerInfo = $dispatcher->dispatch($request);
+            $routerInfo = $dispatcher($request);
```

Comment:
What about `call_user_func($dispatcher, $request)` here?

Penny still requires only PHP 5.4+ and the notation `$dispatcher($request)` will be indipendent only from PHP 7: https://3v4l.org/Rd6qm